src/core/bridge.ts

```diff
@@ -228,6 +228,19 @@ export class BrowserBridge extends AbstractBridge {
   }
 }
 
+export class ProxyBridge extends AbstractBridge {
```

Comment:
rather than add a whole new class, I would simply make `baseUrl` an optional property of `BridgeConfig`
then in the `constructor()`s of `BrowserBridge` and `CloudBridge` you can just have

```js
export interface BridgeConfig {
   timeout?: number;
   baseUrl?: string;
}

// in constructor()

this.baseUrl =  config.baseUrl ||`${scheme}://${customerId}-cors.groupbycloud.com${port}/api/v1`;

// and

this.baseUrl = config.baseUrl || `https://${customerId}.groupbycloud.com:443/api/v1`;
```

Reply:
or even `proxyUrl` so that you don't have to change your other PR much :P